app/src/main/java/com/sebastianvm/mango/model/IncomeInfo.kt

```diff
@@ -0,0 +1,27 @@
+package com.sebastianvm.mango.model
+
+import kotlinx.datetime.LocalDate
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed interface IncomeInfo {
+    val amount: Float
```

Comment:
What do you think about instead using BigDecimal for money?
* we'd have to write value converters and probably a custom serializer too

worried that all the float operations, but we can tackle this later

Reply:
My understanding is that `BigDecimal`, same as `BigInteger`, is an object, which allows the support for large numbers and accuracy, but at the cost of performance. That being said, it is possible that this performance hit won't be felt by the user, so I'm open to trying. Should we do now or after we've done some testing with floats and decide it's necessary.